docs/excel/custom-functions-overview.md

```diff
@@ -374,7 +374,7 @@ The information about a cell's address is exposed only if `requiresAddress` is m
 }
 ```
 
-In the script file (**./src/customfunctions.js** or **./src/customfunctions.ts**), you'll also need to add a `getAddress` function to find a cell's address. This function may take parameters, as shown in the following sample as `parameter1`. The last parameter will always be `invocationContext`, an object containing the cell's location that Excel passes down when `requiresAddress` is marked as `true` in your JSON metadata file.
+In the script file (**./src/functions/customfunctions.js** or **./src/functions/customfunctions.ts**), you'll also need to add a `getAddress` function to find a cell's address. This function may take parameters, as shown in the following sample as `parameter1`. The last parameter will always be `invocationContext`, an object containing the cell's location that Excel passes down when `requiresAddress` is marked as `true` in your JSON metadata file.
```

Comment:
it's just ".src/customfunctions.js" or "./src/customfunctions.ts" The original was correct.